### DISCUSSION TITLE
I can't install cypress recorder

### DISCUSSION THREAD

**Discussion by VALERYRAMIREZ** [cypress-io/cypress#28071] [Questions and Help]
Hi, how are you?.

I have npm version 10.2.0 and cypress  version 13.3.1 and when I try to install cypress recorder with the command 

`npm install cypress-recorder -D`

the terminal gives me the error

`npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/cypress-recorder - Not found
npm ERR! 404
npm ERR! 404  'cypress-recorder@*' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.`

I tried to use yarn add cypress-recorder -D but it didnt work by giving me the same error.

What can I do?

**Comment by portyu9** [comment]
Hi @VALERYRAMIREZ 

The `E404` is because **`cypress-recorder` is not the npm package name**, so switching from npm to Yarn will produce the same result.

If you meant Cypress's Chrome DevTools Recorder converter, the official package is:

```bash
npm install -g @cypress/chrome-recorder
```

or you can run it without a global install:

```bash
npx @cypress/chrome-recorder
```

There is also an official Chrome DevTools Recorder extension that exports recordings as Cypress tests.

For new Cypress projects, another option is **Cypress Studio**, which is built into current Cypress and can record interactions directly while editing an E2E test, so it does not require installing a `cypress-recorder` package.

Therefore the npm/yarn configuration is not the problem here—the requested package name is.

Please mark this answer as accepted if it helped, thank you.
